bin/ud/ud_train.py

```diff
@@ -221,6 +221,13 @@ def evaluate(nlp, text_loc, gold_loc, sys_loc, limit=None):
 
 
 def write_conllu(docs, file_):
+    try:
```

Comment:
We usually try to avoid `try`/`except` blocks, espeically in cases where we "know the answer". 

So maybe we could do something like this instead:

```python
if not Token.has_extension("get_conllu_lines"):
    Token.set_extension("get_conllu_lines", method=get_token_conllu)
# etc.
```

Reply:
Sure!